src/CommunityToolkit.Maui.UnitTests/Views/PopupTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Extensions;
+using CommunityToolkit.Maui.Views;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests;
+
+public class PopupTests : BaseTest
+{
+	readonly IPopup popup = new MockPopup();
+	public PopupTests()
+	{
+		Assert.IsAssignableFrom<IPopup>(new MockPopup());
+		Assert.IsAssignableFrom<IPopup>(new MockBasePopup());
+	}
+
+	[Fact]
+	public void PlaceholderTest()
+	{
+		var page = new ContentPage
+		{
+			Content = new Label
+			{
+				Text = "Hello there"
+			}
+		};
+		var np = new NavigationPage(page);
+		_ = page.Navigation;
+		page.ShowPopup((MockPopup)popup);
```

Comment:
@brminnick This will throw an exception because the `Handler` and `MauiContext` will be `null`. If I'm not wrong, the `Handler` concept doesn't exist on `net6`. That said we can't test the `Popup` because it uses the `Handler`, `Snackbar` and `Toast` don't have a `Handler` implementation then we can test it using `UnitTest`.
I remember that Peppers have a package that mocks `Xamarin.Forms` so you can unit test it, but I can't see something for `Maui`. Do you have any idea on how to workaround that?

Reply:
You'll need to create a mock `ViewHandler` that implements its own `MauiContext` like we did for MockAnimationHandler:

https://github.com/CommunityToolkit/Maui/blob/c4d7d72b1512a187f9ca0d1560e00dcab3fe4543/src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs#L11-L141